cmd/nodereset.go

```diff
@@ -85,7 +85,7 @@ func cleanupNetworking() {
 
 func cleanupDockerImages() {
 	for _, image := range utils.GetImages() {
-		_ = exec.Command("docker", "rmi", "-f", image).Run()
+		_ = exec.Command("docker", "rmi", image).Run()
```

Comment:
I could return an error (with a list of all the images that failed to be removed) and then check for an error where this function is called. If so, it would make sense to do this for the other cleanup functions as well- potentially a separate PR

Reply:
Agreed, errors should be reported

Reply:
+1^